### PR TITLE
Add wait_for_exists_only option to DataVolume wait()

### PR DESCRIPTION
**Pull request by jpeimer**
##### Short description:
Add wait_for_exists_only option to DataVolume wait()

### CODE REVIEW

File: ocp_resources/datavolume.py [hunk 1]
```diff
@@ -341,3 +341,16 @@ def delete(self, wait=False, timeout=TIMEOUT_4MINUTES, body=None):
             return super().delete(wait=wait, timeout=timeout, body=body)
         else:
             return self.pvc.delete(wait=wait, timeout=timeout, body=body)
+
+    def wait_created(self, timeout=TIMEOUT_10SEC, sleep=1):
```
[the remark this answers is not in the document]
Comment (jpeimer) [reply ocp_resources/datavolume.py]: Done

File: ocp_resources/datavolume.py [hunk 2]
```diff
@@ -217,12 +217,15 @@ def wait_deleted(self, timeout=TIMEOUT_4MINUTES):
         super().wait_deleted(timeout=timeout)
         return self.pvc.wait_deleted(timeout=timeout)
 
-    def wait(self, timeout=TIMEOUT_10MINUTES, failure_timeout=TIMEOUT_2MINUTES):
-        self._check_none_pending_status(failure_timeout=failure_timeout)
+    def wait(self, wait_for_exists_only=False, timeout=TIMEOUT_10MINUTES, failure_timeout=TIMEOUT_2MINUTES):
```
Comment (rnetser) [inline ocp_resources/datavolume.py:220]: if someone is using positional args your PR breaks their code; add last

**Review by rnetser — CHANGES_REQUESTED** [review]

File: ocp_resources/datavolume.py (hunk 2 above)
Comment (jpeimer) [reply ocp_resources/datavolume.py]: You're right, thanks

**Review by myakove — APPROVED** [review]

**Comment by redhat-qe-bot2** [comment]
Cherry-picked PR Add wait_for_exists_only option to DataVolume wait() into v4.14

**Pull request merged**